tests/e2e/remix-user-journeys.spec.ts

```diff
@@ -155,36 +157,38 @@ test.describe('Remix user journeys', () => {
   test('response has user-defined Cache-Control header when using origin SSR', async ({ page, serverlessSite }) => {
     const response = await page.goto(`${serverlessSite.url}/headers`)
     await expect(page.getByRole('heading', { name: /Headers/i })).toBeVisible()
-    expect(response?.headers()['cache-control']).toBe('public,max-age=3600')
+    expect(response?.headers()['cache-control']).toBe('public,max-age=3600,durable')
   })
 
   test('response has user-defined Cache-Control header when using edge SSR', async ({ page, edgeSite }) => {
     const response = await page.goto(`${edgeSite.url}/headers`)
     await expect(page.getByRole('heading', { name: /Headers/i })).toBeVisible()
-    expect(response?.headers()['cache-control']).toBe('public,max-age=3600')
+    expect(response?.headers()['cache-control']).toBe('public,max-age=3600,durable')
   })
 
   test('user can configure Stale-while-revalidate when using origin SSR', async ({ page, serverlessSite }) => {
+    const MAX_AGE = 60000 // Must match the max-age set in the fixture
+
     await page.goto(`${serverlessSite.url}/stale-while-revalidate`)
     const responseGeneratedAtText1 = await page.getByText('Response generated at').textContent()
 
-    await page.waitForTimeout(5000)
+    await page.waitForTimeout(MAX_AGE / 2)
 
     await page.goto(`${serverlessSite.url}/stale-while-revalidate`)
     const responseGeneratedAtText2 = await page.getByText('Response generated at').textContent()
     expect(responseGeneratedAtText2, 'First and second response should have matching date and time').toEqual(
       responseGeneratedAtText1,
     )
 
-    await page.waitForTimeout(6000)
+    await page.waitForTimeout(2000 + MAX_AGE / 2)
```

Comment:
Now knowing the true source of the issues, we might be able to revert some of these timing changes